MovingBlocks/TerasologyLauncher
# feat: update command line options for Terasology

Use posix-style command line option syntax for new (picocli-enabled) versions of Terasology.

Fixes #589.

### How to test

The only Terasology command line option we pass is the Terasology home directory, so start games and make sure they're using the data directory specified in the Launcher settings.

### Outstanding before merging

- [x] handle the mac-specific `noSplash`
- [x] detect Terasology version and behave appropriately
- [ ] reinstalls game after starting launcher? https://github.com/MovingBlocks/TerasologyLauncher/pull/646#issuecomment-894854510
